Classes/Controller/GndController.php

```diff
@@ -0,0 +1,79 @@
+<?php
+namespace EWW\Dpf\Controller;
+
+/*
+ * This file is part of the TYPO3 CMS project.
+ *
+ * It is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License, either version 2
+ * of the License, or any later version.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE.txt file that was distributed with this source code.
+ *
+ * The TYPO3 project - inspiring people to share!
+ */
+
+
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+
+/**
+ * GndController
+ */
+class GndController extends \EWW\Dpf\Controller\AbstractController
+{
+    // TODO: Make gndHost and Urls configurable
```

Comment:
Would be nice, but we can live with this hardcoded urls for now.

Reply:
How much effort could it be to make it configurable? There is a configuration mechanism in place already...